tests/decorators/test_decorators.py

```diff
@@ -0,0 +1,802 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+import unittest
+
+from azure.functions.decorators.core import DataType, AuthLevel, \
+    BindingDirection, AccessRights, Cardinality
+from azure.functions.decorators.function_app import FunctionApp
+from azure.functions.decorators._http import HttpTrigger, HttpMethod
+from azure.functions.decorators.timer import TimerTrigger
+from tests.decorators.testutils import assert_json, get_func
+
+
+class TestFunctionsApp(unittest.TestCase):
+    def setUp(self):
+        self.func_app = FunctionApp()
+
+    def test_route_is_function_name(self):
+        app = self.func_app
+        test_func_name = "dummy_function"
+
+        @app.function_name(test_func_name)
+        @app.route()
+        def dummy_func():
+            pass
+
+        func = get_func(self, app)
```

Comment:
If `get_func` is only used in this module  - no need to put it in testutils as its utility is limited to one module, and it'll be better that the definition is available for reference.